package.json

```diff
@@ -58,6 +59,7 @@
   },
   "dependencies": {
     "lodash.isequal": "^4.5.0",
-    "object.entries": "^1.0.4"
+    "object.entries": "^1.0.4",
+    "preact-render-to-string": "^3.6.3"
```

Comment:
was just thinking, we could probably peer / optional dep this since it's only used for the snapshot renderer?